config/settings.yml

```diff
@@ -45,3 +45,7 @@
         :ems_refresh_worker_amazon_network: {}
         :ems_refresh_worker_amazon_ebs_storage: {}
         :ems_refresh_worker_amazon_s3: {}
+:blacklisted_event_names:
```

Comment:
@bronaghs sorry for the confusion - of course we want this editable. But the toplevel key `:blacklisted_event_names:` would be overwritten or overwrite the same key in the core repo or other providers - because the arrays under the key are not merged, but overwritten.

Thats why I think it should be under the `:amazon:` top level key namespace